packages/design-system-docs/src/pages/startup/child-design-systems.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Child Design Systems
+weight: 6
+---
+
+The CMS design system (CMSDS) provides a set of components and styles to meet many design and development needs. However, there are cases where product teams need to modify a design system beyond [theming]({{root}}/startup/theming/) and publish the new system for other teams to consume.
+
+## Who should use a child design system?
+
+A common use case for child design systems is when multiple teams are working under the same CMS site. These teams would need to share custom assets and code (i.e. brand colors, site header, custom icons, etc), and would benefit from a documentation site to reference as single source of truth. If you are working as a single team and don't have a need for a documentation site, a child design system is probably not for you.
```

Comment:
Consider updating to 

A common use case for child design systems is when multiple product teams are working under the same CMS site. These separate product teams would share custom assets and code (i.e. brand colors, site header, custom icons, etc), and would benefit from a documentation site to reference as a single source of truth. If you are working as a single team and don't have a need for a documentation site, a child design system is probably not for you.